torch_glow/tests/utils.py

```diff
@@ -7,6 +7,35 @@
 SUBGRAPH_ATTR = "Subgraph"
 
 
+def get_image(filename):
```

Comment:
For this chunk of code, I have already checked with Adnan, and he confirmed it should be Okay to publish this to OSS.
In order to repro this with real-life img, this function is necessary.